srml/fees/src/lib.rs

```diff
@@ -14,7 +14,114 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Handles all transaction fee related operations
+//! ## Overview
+//! 
+//! The `fees` module handles transaction fee related operations. To use this module, the [`TransferAsset`](https://crates.parity.io/srml_fees/trait.Trait.html#associatedtype.TransferAsset) trait must be implemented to enable asset transfer between accounts. This module provides the following functionality
+//! 
+//! - Getting and setting the base and per-byte transaction fee
+//! - Tracking the overall charged fees for each transaction
+//! - Charging and refunding transaction fees
+//! 
+//! ## Interface
+//! 
+//! ### Dispatchable
+//! 
+//! This module is intended to be called by other modules rather than transactions (no dispatchable functions).
+//! 
+//! ### Public
+//! 
+//! The following methods are publicly accessible to other modules that implement the [`ChargeFee`](https://github.com/paritytech/substrate/blob/master/srml/fees/src/lib.rs#L93) trait.
```

Comment:
well no other module doesn't implement ChargeFee. They access to this using an associated type that implements it (so an associated module in a way) or by being bound to srml_fees.